docker-compose-lviv.yml

```diff
@@ -32,6 +32,8 @@ services:
       - db
     networks:
       - sip
+    env_file:
+      - .env
```

Comment:
Include it in the build args, the same way as e.g. MAPBOX_ACCESS_TOKEN

Reply:
done